ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/liujiayi771/velox.git
+VELOX_BRANCH=stddev_pop
```

Comment:
Same.

Reply:
I have reverted these changes. By the way, I will support the two functions var_samp and var_pop later.